Android/app/src/main/java/hr/foi/air/fitfusion/TrainingSessionActivity.kt

```diff
@@ -0,0 +1,111 @@
+package hr.foi.air.fitfusion
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.ArrayAdapter
+import android.widget.Button
+import android.widget.EditText
+import android.widget.Spinner
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.database.DatabaseReference
+import com.google.firebase.database.FirebaseDatabase
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.*
+import hr.foi.air.fitfusion.data_classes.TrainingModel
+
+class TrainingSessionActivity : AppCompatActivity() {
+    private lateinit var etTime: EditText
+    private lateinit var etDate: EditText
+    private lateinit var etNumber: EditText
+    private lateinit var sspinner: Spinner
+    private lateinit var btnDone: Button
+    private lateinit var btnCancel: Button
+    private lateinit var databaseRf: DatabaseReference
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.new_training_session)
+
+        etTime = findViewById(R.id.Time)
+        etDate = findViewById(R.id.Date)
+        etNumber = findViewById(R.id.Number)
+        sspinner = findViewById(R.id.spinner)
+        btnDone = findViewById(R.id.Done)
+        btnCancel = findViewById(R.id.Cancel)
+
+        val classNames = listOf(
+            "Yoga",
+            "Strength",
+            "Cardio"
+        )
+
+        val adapter = ArrayAdapter(this, android.R.layout.simple_spinner_item, classNames)
+        adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item)
+        sspinner.adapter = adapter
+
+        databaseRf = FirebaseDatabase.getInstance().getReference("Training")
```

Comment:
Can you please put the part of your code that works with database in FirebaseManager so we can work with three layered architecture